modules/hitlnext/src/views/full/app/components/HandoffListHeader.tsx

```diff
@@ -89,11 +99,62 @@ const HandoffListHeader: FC<Props> = ({ filterOptions, sortOption, setFilterOpti
     }
   ]
 
+  const renderButtons = (tags): ToolbarButtonProps[] => {
```

Comment:
Add typings like so:

```suggestion
  const renderButtons = (tags: string[]): ToolbarButtonProps[] => {
```

or 

```suggestion
  type Tags = string[]
  // ...
  const renderButtons = (tags: Tags): ToolbarButtonProps[] => {
```